python/paddle/jit/sot/opcode_translator/executor/function_graph.py

```diff
@@ -906,11 +905,11 @@ def collect_related_dummy_tensor(var):
             if isinstance(
                 output.tracker, (DummyTracker, SymbolicOperationTracker)
             ):
-                if is_graph_output(output):
+                if self.is_graph_output(output):
```

Comment:
```suggestion
                if FunctionGraph.is_graph_output(output):
```

明确标明是 `staticmethod` 吧